src/main/java/net/minecraftforge/fml/common/registry/VillagerRegistry.java

```diff
@@ -151,33 +208,161 @@ private void init()
         VillagerProfession prof = new VillagerProfession("minecraft:farmer", "minecraft:textures/entity/villager/farmer.png");
         {
             register(prof, 0);
-            (new VillagerCareer(prof, "farmer")).init(VanillaTrades.trades[0][0]);
```

Comment:
This entire section: No
The original arrays are in the format below because they are more easily diffed against the original arrays during MC updates to make sure we did not miss anything. The init functions and trades arrays are kept private because modders should NOT be accessing them .